client/src/components/CategoriesChart.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { PieChart, Pie, Cell, Legend } from 'recharts';
+
+CategoriesChart.propTypes = {
+  categories: PropTypes.object,
+};
+
+export default function CategoriesChart({ categories }) {
+  const data = [];
+  const labels = Object.keys(categories);
+  const values = Object.values(categories);
+  for (let i = 0; i < labels.length; i++) {
+    data.push({ name: labels[i], value: Math.round(values[i]) });
+  }
+
+  // threads colors (TO DO:  USE THREADS VARIABLE NAMES!!!)
+  const COLORS = ['#0868b8', '#2ab589', '#d9bc2d', '#d44648'];
```

Comment:
you can import these like so

```tsx
import colors from 'plaid-threads/scss/colors';

...
const chartColors = [colors.red800, colors.blue800, ...

```

Reply:
Thank you...I was going to ask you how to do this...